src/components/Arrow.js

```diff
@@ -16,15 +16,19 @@ function Arrow ({
 {
 	theme,
 }) {
-	const classes = StyleSheet.create(deepMerge(defaultStyles, theme));
-
 	return (
 		<button
 			type="button"
-			className={css(classes.arrow, classes['arrow__direction__' + direction], size && classes['arrow__size__' + size])}
+			className={`arrow ${styles.arrow} ${styles[`arrow__size__${size}`]}`}
 			onClick={onClick}
 			onTouchEnd={onClick}
-			{...props}
+			style={{
```

Comment:
Why have this here and not in css ?

Reply:
These are remnants of the _old aphrodite way_, they are dynamically overridable by providing a `theme={theme}` object to the gallery.

Of course they may be factored out, but I suggest scoping it out to another, let's call it a _cleanup_ story if need be.